transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/AddTransferMethodFragment.java

```diff
@@ -457,6 +458,21 @@ public void showInputErrors(@NonNull final List<HyperwalletError> errors) {
                             widget.getView(mDynamicContainer).requestFocus();
                             focusSet = true;
                         }
+                        HyperwalletInsight.getInstance().trackError(requireContext(),
+                                HyperwalletInsight.PAGE_TRANSFER_METHOD_COLLECT_ACCOUNT,
+                                HyperwalletInsight.TRANSFER_METHOD_GROUP,
+                                new ErrorInfo.ErrorInfoBuilder().type(HyperwalletInsight.ERROR_TYPE_API)
+                                        .code(error.getCode())
+                                        .message(error.getMessage())
+                                        .field(error.getFieldName())
+                                        .params(new HyperwalletInsight.TransferMethodParamsBuilder()
```

Comment:
Sorry about this confusion, we decided to remove this implementation in `params`, please remove the set `params` on this case

Reply:
I have removed the set params for this case and pushed